xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2373,7 +2373,7 @@ xe.admin.configurable.title=Custom configurable sections
 xe.admin.configurable.macros.title=Macros for custom configurable sections
 xe.admin.configurable.noPermissionThisApplication=You don't have permission to configure this application.
 xe.admin.configurable.applicationAuthorNoAdmin=This configuration cannot be displayed because it was last edited by [[{0}]] who doesn''t have permission to edit this page.
-xe.admin.configurable.cannotLockNoJavascript=This page cannot be locked for editing because Javascript is turned off, for page editing safety, please enable Javascript.
+xe.admin.configurable.cannotLockNoJavascript=This page cannot be locked for editing because Javascript is turned off.For page editing safety, please enable Javascript.
```

Comment:
You're missing a space before the "For". 

Also note that this text has already been changed by another student. FTR I had already commented that I didn't like to have long sentences but it was still committed apparently. Funnily the suggestion from this student is to put it back as it was previously ;)